src/webServer/webServer.ts

```diff
@@ -0,0 +1,193 @@
+/*@internal*/
+namespace ts.server {

```

Comment:
I don't understand the split between the two webserver.ts files.  Can you please explain?

Reply:
webServer/webserver is common logic that can be used by our test harness so that we have tests when only few system functionality is implemented and how webserver session works with that. 
tsserver/webServer.ts is counter part of tsserve.js-> nodeServer(the current one which specializes tsserver.js specific changes) eg setting ts.sys to web system etc.